polyphemus/lib/etls/stub_metis_file_etl.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../metis_file_etl'
+
+class Polyphemus::StubMetisFileEtl < Polyphemus::MetisFileEtl
+  def initialize
```

Comment:
To replace with other etls (such as the new bulk rna linker, etc).  This is a test etl.